### PR TITLE
Oct

### PR DESCRIPTION
**Pull request by EngOmarAhmedIOS**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ Check ] All writings are my own.
- [ Check ] My entry follows the Codecademy Docs style guide.
- [ Check ] My changes generate no new warnings.
- [ Check ] I have performed a self-review of my own writing and code.
- [ Check ] I have checked my entry and corrected any misspellings.
- [ Check ] I have made corresponding changes to the documentation if needed.
- [ Check ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ Check ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ Check ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by EngOmarAhmedIOS** [comment]
can you please review my work ?

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:69]:
````suggestion
```shell
````

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:48]:
````suggestion
```shell
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@EngOmarAhmedIOS I did a first review and left a few comments.

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (EngOmarAhmedIOS) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:27]: you want me to delete the parameters header

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (EngOmarAhmedIOS) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:54]: I don't get what is the change I should do, I don't see any difference

Comment (SSwiniarski) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:52]: @EngOmarAhmedIOS There's a line break after the header.

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [reply content/python/concepts/built-in-functions/terms/oct/oct.md:26]: Yes

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (EngOmarAhmedIOS) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:7]:
```suggestion
  - 'Computer Science'
  - 'Data Science' 
  - 'Code Foundations'
```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (EngOmarAhmedIOS) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:12]:
```suggestion
  - 'Numbers'
  - 'Data Types'
```

**Comment by EngOmarAhmedIOS** [comment]
I corrected all the requested changes. please review the final file @SSwiniarski

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:3]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:5]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:34]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:41]:
```suggestion
```
Remove extra line break.

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:72]:
```suggestion
```
Remove extra line break.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@EngOmarAhmedIOS I had a few more comments, all about removing extra line breaks in the entry.

**Review by SSwiniarski — APPROVED** [review]
@EngOmarAhmedIOS Looks good for a second review.

**Comment by EngOmarAhmedIOS** [comment]
Now I am officially contributor with CodeAcadmey Docs or Should I wait for a second review before celebrating this moment 😁 @SSwiniarski

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:17]:
```suggestion
The **`oct()`** function takes an integer and returns the corresponding octal string.

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:24]:
```suggestion
Where `integer` is the value to be converted into an octal string.

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:51]:
```suggestion

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:30]:
```suggestion
This example returns the octal string for the value `10`.

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:56]:
```suggestion
# Binary number  

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:58]:
```suggestion
# Hexadecimal number  

```

File: content/python/concepts/built-in-functions/terms/oct/oct.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/oct/oct.md:48]:
```suggestion
In Python, binary and hexadecimal values are integer values. So, the `oct()` function can convert binary and hexadecimal values as well. 

```

**Review by caupolicandiaz — APPROVED** [review]
@EngOmarAhmedIOS @SSwiniarski Looks good. I just removed a couple of extra lines (there should never be multiple lines between content) and I decided to change the second example to a codebyte.

**Comment by EngOmarAhmedIOS** [comment]
> @EngOmarAhmedIOS @SSwiniarski Looks good. I just removed a couple of extra lines (there should never be multiple lines between content) and I decided to change the second example to a codebyte.

@caupolicandiaz is there anything I should do now

**Comment by EngOmarAhmedIOS** [comment]
Should I celebrate 🎉 this moment and consider my self as a contributor with Codeacadmey @caupolicandiaz 😁😁

**Comment by SSwiniarski** [comment]
@EngOmarAhmedIOS Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/python/built-in-functions/oct

image

**Pull request merged**
